.github/workflows/ci.yml

```
@@ -2,7 +2,7 @@ name: CI
 
 on:
   push:
-    branches: [ "ci", "dev", "build*" ]
+    branches: [ "ci", "dev", "build*", "implementation*" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)